docs/conf.py

```diff
@@ -156,6 +158,44 @@
     "dudir": ("http://docutils.sourceforge.net/docs/ref/rst/directives.html#%s", ""),
 }
 
+clang_format_signatures_domain_styles = {
+    "cpp": """{
+    BasedOnStyle: LLVM,
+    ColumnLimit: 70,
+    }""",
+}
+
+html_wrap_signatures_with_css = ["py"]
+
+external_cpp_references = {
```

Comment:
If you prepend comments here that describe which extension this option is related to, then it would be easier to use a `literalinclude` block in the docs.

As an example, I did this for the `kbd_keys` extension's `keys_map` config option.

Reply:
Done.